app/models/flood_risk_engine/address_search.rb

```diff
@@ -0,0 +1,5 @@
+module FloodRiskEngine
+  class AddressSearch < ActiveRecord::Base
```

Comment:
PENDING - RIp-74 does not specify exact behaviour of back button/link
 +        # If we clear address_search now, when they go back the drop down
 +        # is cleared and shows zero results. But not sure what the designers want displayed once an address
 +        # has been saved. Maybe the dropdown should disappear and the address fields be displayed
 +        # enrollment.address_search.delete if(result)